dDatabase/GameDatabase/CMakeLists.txt

```diff
@@ -31,6 +38,8 @@ file(
 	ITables/*.h
 )
 
+target_compile_definitions(dDatabaseGame PRIVATE DLU_SQLITE_DATABASE=${DLU_SQLITE_DATABASE} DLU_MYSQL_DATABASE=${DLU_MYSQL_DATABASE})
```

Comment:
pretty sure you removed this feature since it'll always compile both?
can this be removed as well?

Reply:
yep